README.md

```diff
@@ -2,391 +1,310 @@
 
+There is some additional code in `index.jsp`.
 
-JSONObject response = dataApi.requestJSONObject();
-JSONObject res = new JSONObject(response.getString("body"));
+Our web app is configured to start in this block in `web.xml`:
 
-// You can also get a Remote object like this:
-Remote remote = dataApi.request();
-String body = remote.getBody();
+``` XML
+<welcome-file-list>
+    <welcome-file>index.jsp</welcome-file>
+</welcome-file-list>
 ```
 
-## Troubleshooting
+This marks the end of the quick start guide. From here, try modifying the example files yourself, you are welcome to use this code as a basis for your own projects.
 
-We recommend that you use the Java JSON implementation which comes packaged in the sdk. In the past, we had several issues which were due to incompatible JSON implementations.
+Take a look at some more in-depth options and tutorials on using Learnosity assessment functionality below.
 
+[(Back to top)](#table-of-contents)
 
-## Creating jar files (using Eclipse Luna Release 4.4.0)
+## Next steps: additional documentation
 
-The 'light' jar:
+### **SDK reference**
+See a more detailed breakdown of all the SDK features, and examples of how to use more advanced or specialised features on the [SDK reference page](REFERENCE.md).
 
-In eclipse right click the java sdk project and select 'Export'. Under 'Java', select 'JAR file'. In the following screen only select the packages in the src (excluding the library folder) and click 'Finish'.
+### **Additional quick start guides**
+There are more quick start guides, going beyond the initial quick start topic of loading an assessment, these further tutorials show how to set up authoring and analytics:
+* [Authoring Items quick start guide](https://help.learnosity.com/hc/en-us/articles/360000754958-Getting-Started-With-the-Author-API) (Author API) - create and edit new Questions and Items for your Item bank, then group your assessment Items into Activities, and
+* [Analytics / student reporting quick start guide](https://help.learnosity.com/hc/en-us/articles/360000755838-Getting-Started-With-the-Reports-API) (Reports API) - view the results and scores from an assessment Activity. 
 
-The 'all' jar:
+### **Learnosity demos repository**
+On our [demo site](https://demos.learnosity.com/), browse through many examples of Learnosity API integration. You can also download the entire demo site source code, the code for any single demo, or browse the codebase directly on GitHub.
 
-In eclipse right click the java sdk project and select 'Export'. Under 'Java', select 'Runnable JAR file', click 'Next' and then 'Finish'. There are warnings that some libararies are included. This can be disregarded as the libraries are publicly available.
+### **Learnosity reference documentation**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
### **Learnosity reference documentation**

```